pyresample/area_config.py

```diff
@@ -469,11 +472,10 @@ def _make_area(area_id, description, proj_id, proj_dict, shape, area_extent, **k
         height, width = shape
     if None not in (area_extent, shape):
         return AreaDefinition(area_id, description, proj_id, proj_dict, width, height, area_extent, **kwargs)
-    elif area_extent is not None or shape is not None:
-        return DynamicAreaDefinition(area_id=area_id, description=description, projection=proj_dict, width=width,
-                                     height=height, area_extent=area_extent, rotation=kwargs.get('rotation'),
-                                     optimize_projection=optimize_projection)
-    raise ValueError('Not enough information provided to create an area definition')
+
```

Comment:
I think this `if` was deliberate. I'm ok changing it if it works, but basically this is supposed to catch the user not providing enough information to even make a proper DynamicAreaDefinition. What does DAD do if `area_extent` or `shape` are `None`?

Reply:
The area extent is computed from the data if missing. The shape, if missing, tries to preserve the resolution.

Reply:
ok i'll believe you 😄